### ISSUE TITLE
Rework the travis environment

### ISSUE THREAD

**Issue by mithro** [timvideos/gst-switch#56]
Currently the travis setup isn't quite clear. See discussion at http://logs.timvideos.us/%23timvideos/%23timvideos.2014-12-01.log.html#t2014-12-01T06:06:42

```
mithro	hyades: then for the Python API we have style, unittest and integration
mithro	hyades: and for gst-switch we have style, unittest, integration
mithro	currently gst-switch has "empty" unittest suite
mithro	the style ones just check for style compliance
mithro	and don't run any tests or anything
```

**Comment by mithro** [comment]
I started work at https://github.com/mithro/gst-switch/tree/travisfix but wasn't able to figure out why coverage stuff wasn't working.

**Comment by mithro** [comment]
Fixed in #185
